api/mesh/v1alpha1/meshgateway/schema.yaml

```diff
@@ -0,0 +1,140 @@
+properties:
+  labels:
+    additionalProperties:
+      type: string
+    type: object
+  name:
+    type: string
```

Comment:
Now you are missing mesh so seems like a condition may be reversed :)

Reply:
it's on lines https://github.com/kumahq/kuma/pull/12272/files#diff-17ba8fbac8021b72549d3c48704eee402f7ca764a8e409622ea5f24715e7c227R6-R7